app/uk/gov/hmrc/customs/emailfrontend/services/EmailCacheService.scala

```diff
@@ -60,19 +61,33 @@ class EmailCacheService @Inject()(caching: Save4LaterCachingConfig, applicationC
   }
 
   def saveTimeStamp(internalId: InternalId, verifiedEmailTimestamp: DateTime)
-               (implicit hc: HeaderCarrier, executionContext: ExecutionContext): Future[CacheMap] = {
+                   (implicit hc: HeaderCarrier, executionContext: ExecutionContext): Future[CacheMap] = {
+
     import uk.gov.hmrc.customs.emailfrontend.DateTimeUtil._
     Logger.info("saving verified email time stamp to save 4 later")
     cache[DateTime](internalId.id, timestampKey, verifiedEmailTimestamp)
   }
 
+  private def fetchTimeStamp(internalId: InternalId)(implicit hc: HeaderCarrier, executionContext: ExecutionContext): Future[Option[DateTime]] = {
+    import uk.gov.hmrc.customs.emailfrontend.DateTimeUtil._
+    Logger.info("calling save 4 later to retrieve timestamp")
```

Comment:
for consistency with other loggers in this class, text could be  `retrieving cached timestamp from save 4 later`